src/System.Management.Automation/engine/remoting/commands/InvokeCommandCommand.cs

```diff
@@ -263,6 +263,46 @@ public override SwitchParameter UseSSL
             }
         }
 
+        internal class ValidateVersionOffAttribute : ValidateArgumentsAttribute
+        {
+            protected override void Validate(object arguments, EngineIntrinsics engineIntrinsics)
+            {
+                Version version = arguments as Version;
+                Version s_offVersion = new Version(0, 0);
```

Comment:
Please make `s_offVersion` a static field.  This can be added at line 265 above:
```csharp
private static readonly Version s_OffVersion = new Version(0, 0);
```